pkg/resourcemapping/operator_resource_mapping.go

```diff
@@ -235,13 +248,35 @@ func (ormClient *ORMClient) populateORMTemplateMap(ormCR unstructured.Unstructur
 	return ormTemplateMap, nil
 }
 
-func (ormClient *ORMClient) retrieveORM(ownedObj *unstructured.Unstructured, ownerReference discoveryutil.OwnerInfo) (*ORMSpec, *unstructured.Unstructured, error) {
+func createV1OwnedResourcePath(owned []*devopsv1alpha1.ResourcePath, ownedObj *unstructured.Unstructured) map[string][]devopsv1alpha1.ResourcePath {
```

Comment:
nit, rename to something like defaultResourcePathWithOwned(..) to clarify that the default ResourcePath returned in case of any errors is one containing the owned object.